client/db.go

```diff
@@ -99,6 +99,23 @@ type DB interface {
 	//
 	// Note: A policy can not be added without the creatorID (identity).
 	AddPolicy(ctx context.Context, policy string) (AddPolicyResult, error)
+
+	// AddDocActorRelationship creates a relationship between document and the target actor.
+	//
+	// Moreover, since collection has the policy and resource information we don't need to
+	// explicitly have the user provide them as arguments.
```

Comment:
> // Moreover, since collection has the policy and resource information we don't need to
	// explicitly have the user provide them as arguments

todo: I'm not sure I understand this, can you please reword it?  Are some of these params optional?

Reply:
params are not optional was trying to explain that the magic of having policy and resource info on collection is nice because the user doesn't have to input that but just the collection name. going to remove it as it causes more confusion that clarification

- [x] extra fluff makes things more confusing to remove